process/process.go

```diff
@@ -11,16 +11,17 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strconv"
 	"strings"
 	"sync"
 
+	"github.com/sirupsen/logrus"
```

Comment:
For consistency with the rest of the code

```suggestion
	log "github.com/sirupsen/logrus"
```